### PR TITLE
Add firma support

### PR DESCRIPTION
**Pull request by micah**
Firma is an encrypted mailing list manager: https://firma.fluxo.info/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,6 +171,10 @@ Default: False.
 A Boolean to define whether to configure master.cf to use dovecot as the local delivery agent.  
 Default: False.
 
+#### `use_firma`####
```
Comment (mcanevet) [inline README.md]: Missing `#`

Comment (micah) [inline README.md]: also it has trailing ####, which is not how the other entries have. I've fixed this and am pushing a new commit here

Comment (micah) [inline README.md]: ping?

File: spec/classes/postfix_spec.rb [hunk 2]
```diff
@@ -84,6 +84,7 @@
               :root_mail_recipient => 'foo',
               :use_amavisd         => true,
               :use_dovecot_lda     => true,
+              :use_firma           => true,
```
Comment (raphink) [inline spec/classes/postfix_spec.rb]: Shouldn't this be false in the general testing case?

**Comment by raphink** [comment]
@micah are you still interested in getting this merged?

**Comment by raphink** [comment]
Closing this for now. Please reopen if you wish to rebase and fix this PR.

**Comment by micah** [comment]
No, i think that this can be closed now. As far as I can tell, firma is
no longer an active project.

**Comment by raphink** [comment]
Thanks.

**Pull request closed without merging**